docs/_custom-rule-tutorial-code/eslint-plugin-foo-bar.js

```diff
@@ -0,0 +1,4 @@
+"use strict";
```

Comment:
Because we are using these as examples, I think it would be helpful to have a `@fileoverview` comment at the top explaining what each file does. Most of the files in the eslint repo already have this, so you can copy-paste from there.

Reply:
> Because we are using these as examples, I think it would be helpful to have a `@fileoverview` comment at the top explaining what each file does. Most of the files in the eslint repo already have this, so you can copy-paste from there.

I like this suggestion.